src/main/java/formflow/library/FormFlowController.java

```diff
@@ -41,10 +45,23 @@ public abstract class FormFlowController {
     this.messageSource = messageSource;
   }
 
+  /**
+   * Saves a {@link Submission}
+   *
+   * @param submission {@link Submission} saved through the submission repository service.
+   * @return The saved {@link Submission} object
+   */
   protected Submission saveToRepository(Submission submission) {
     return saveToRepository(submission, null);
   }
 
+  /**
+   * Saves a {@link Submission}
+   *
+   * @param submission  - a submission saved through the submission repository service
```

Comment:
I just noticed that this javadoc set has `-` after the param name, which isn't necessary.  It results in weird rendering. 
<img width="742" alt="Screenshot 2024-01-12 at 9 38 00 AM" src="https://github.com/codeforamerica/form-flow/assets/7966375/a4fcb6e9-ec8d-4311-8dce-9779176658a1">